docs/src/pages/Conformance.md

```diff
@@ -272,8 +272,14 @@ The IBM FHIR Server does not consider the `Quantity.comparator` field as part of
 ### Searching on URI
 URI searches on the IBM FHIR Server are case-sensitive with "exact-match" semantics. The `above` and `below` prefixes can be used to perform path-based matching that is based on the `/` delimiter.
 
+There is one exception to the statement above. The `url` search parameter, which is defined in the base FHIR specification on definitional resource types as a URI search parameter, is actually treated as a canonical search parameter of type REFERENCE, as documented in the [FHIR specification](http://build.fhir.org/references.html#canonical). The following section of this document describes how the IBM FHIR Server processes canonical reference searches.
```

Comment:
Unless we're dependent on some updated text, I usually try to scope spec links to R4 vs the latest draft.

Reply:
Done.